cjs/interface/event-target.js

```diff
@@ -15,24 +54,43 @@ class DOMEventTarget extends EventTarget {
     return null;
   }
 
+  addEventListener(type, listener, options) {
+    const map = wm.get(this);
+    if (!map.has(type)) 
+      map.set(type, new Map);
+    const listeners = map.get(type).set(listener, options);
+    wm.set(this, map.set(type, listeners));
+  }
+
+  removeEventListener(type, listener) {
+    const map = wm.get(this);
+    if (map.has(type)) {
+      const listeners = map.get(type);
+      if (listeners.delete(listener) && !listeners.size) {
+        map.delete(type);
+      } else {
+        map.set(type, listeners);
+      }
+      wm.set(this, map);
```

Comment:
I am confused ... why are you setting the `this` every time to the same map it had already?

Reply:
Good point, I had an error because i forgot a genuily usefull `set`, but overdid the "fix" forgetting we store the reference anyway...

Reply:
to be honest I don't understand neither why you are setting anything on removeEventListener ... if it's not there, don't set it?

Reply:
All those `set` should be removed now